<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting_window</class>
 <widget class="QWidget" name="setting_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QTabWidget" name="gpt_setting">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>gpt语音</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>312</width>
              <height>447</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>API BASE</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_edit_api_base"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>API KEY</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_edit_api_key"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>上下文数量</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_box_context_cnt">
                 <property name="value">
                  <number>20</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>GPT语速</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spin_box_speak_speed">
                 <property name="value">
                  <number>9</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>语音转文本模式</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combo_box_s2t_mode">
                 <item>
                  <property name="text">
                   <string>本地</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>远程</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>语音转文本远程地址</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_edit_s2t_remote_address"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>语音生成模式</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combox_box_t2s_mode">
                 <item>
                  <property name="text">
                   <string>本地</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>远程</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>语音生成远程地址</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="line_edit_t2s_remote_address"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="btn_gpt_setting_confirm">
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_gpt_setting_cancel">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="subtitle_set">
        <attribute name="title">
         <string>实时字幕</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>312</width>
              <height>447</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>采样时间</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="subtitle_sample_time">
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>模式</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="subtitle_mode_select_combo">
                 <item>
                  <property name="text">
                   <string>本地</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>远程</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>远程地址</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="subtitle_remote_address"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="subtitle_confirm">
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="subtitle_setting_cancel">
             <property name="text">
              <string>取消</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
